var/spack/repos/builtin/packages/qoz/package.py

```diff
@@ -12,6 +12,8 @@ class Qoz(CMakePackage):
     git = "https://github.com/robertu94/QoZ"
     homepage = git
```

Comment:
FWIW. I don't think spack recognizes `git` as a valid homepage *but* we have few tools that actually check it.